packages/storybook/stories/CustomIcons/utils.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { mergeStyles, Stack, TextField } from '@fluentui/react';
+import React, { useState } from 'react';
+
+const iconGrid = (allIcons: Record<string, JSX.Element>, searchText?: string): JSX.Element[] => {
```

Comment:
nit: this makes sense to be a react element
```suggestion
const IconGrid = (props: { allIcons: Record<string, JSX.Element>, searchText?: string }): JSX.Element[] => {
```

then usage is 
```
<IconGrid icons={icons} searchText={searchText} />
```

Reply:
Though if I'm being super nitty the search function would be better as an extracted fn on its own and do
```
<IconGrid icons={filterIcons(icons, searchText)} />
```
But I won't block on that :D

Reply:
Although this in its current form isn't a "grid" as the name suggestions - its just an array of icons. Would be best to move the `<div style={{ display: 'grid', gridTemplateColumns: 'auto auto auto', padding: '1rem 0' }}>` into here and have
```
<Stack>
  <SearchBar onChange={(val) => setSearchText(val)} />
  <IconGrid icons={filterIcons(icons, searchText)} />
</Stack>
```
below

Reply:
I realised the flaws. Fixed everything in the refactor. Take a look.